### DISCUSSION TITLE
Reset Password Token: Invalidate

### DISCUSSION THREAD

**Discussion by msassmann** [fastapi-users/fastapi-users#1146] [Q&A] [answered]
Hello Community!

Is it possible to invalidate the reset password token after it was successfully used?

Thank you very much in advance

**Comment by frankie567** [comment, marked as answer]
Hi @msassmann 👋 

That was something I had in mind for a while, and your message gave me the right nudge to finally tackle this 😄 Reset password tokens are now single-use in the latest version: https://github.com/fastapi-users/fastapi-users/releases/tag/v10.3.0

**Reply by msassmann** [reply]
Hi @frankie567 

Thank you very much!
